themes/tiddlywiki/vanilla/sidebar-slider.tid

```diff
@@ -0,0 +1,90 @@
+title: $:/themes/tiddlywiki/vanilla/sidebar-slider
+tags: $:/tags/Stylesheet
+
+\define sidebarbreakpoint() <$text text={{$:/themes/tiddlywiki/vanilla/metrics/sidebarbreakpoint}}/>
+\define sidebarbreakpoint-minus-one() <$text text={{{ [{$:/themes/tiddlywiki/vanilla/metrics/sidebarbreakpoint}removesuffix[px]subtract[1]addsuffix[px]] ~[{$:/themes/tiddlywiki/vanilla/metrics/sidebarbreakpoint}] }}}/>
+
+<$list filter="[[$:/state/sidebar]is[missing]] [{$:/state/sidebar}match[yes]] +[limit[1]]" emptyMessage="""
+
+	.tc-sidebar-slider {
+		display: none;
+	}
+
+""">
+
+	@media (min-width: <<sidebarbreakpoint>>) {
+
+		.tc-sidebar-slider {
+			position: fixed;
+			width: 42px;
+			top: 0;
+			height: 100%;
+			cursor: col-resize;
+		}
+
+		<$list filter="[{$:/themes/tiddlywiki/vanilla/options/sidebarlayout}match[fluid-fixed]]">
+
+			.tc-sidebar-slider {
+				left: calc(100% - {{$:/themes/tiddlywiki/vanilla/metrics/sidebarwidth}});
+				left: clamp({{$:/themes/tiddlywiki/vanilla/metrics/storywidth}}, calc({{$:/state/sidebar-slider!!event-fromcatcher-posx}}px - {{$:/state/sidebar-slider!!slider-event-fromselected-posx}}px), calc(100% - {{$:/themes/tiddlywiki/vanilla/metrics/sidebarwidth}}));
```

Comment:
state vars are missing if they have never been initialized.

Reply:
yes they are missing in the beginning, that's why the first line gets preferred over the invalid line